.gitignore

```diff
@@ -4,6 +4,6 @@ __pycache__/
 hc.sqlite
 hc/local_settings.py
 static-collected
-
+static/
```

Comment:
You can learn more about static file management in Django here https://docs.djangoproject.com/en/1.10/howto/static-files/

Also, we mentioned WhiteNoise as a Django middleware that does this function even better. See http://whitenoise.evans.io/en/stable/django.html#make-sure-staticfiles-is-configured-correctly